### PR TITLE
MRG: More PSD modes

### LINKED ISSUE #3631: ENH: example with noise levels from empty room
People often show noise levels of MEG systems as e.g. 10 fT/Hz**0.5. We should probably have an example where we do this with data. I'm thinking it would be cool to have it for empty-room data from different sites and/or systems, too, maybe eventually with e.g. sythetic gradiometry and/or compensation grades for systems that support those.

I thought about showing how different denoising algorithms work for some of these datasets, but that might make the example too dense. But collecting these files would at least make it more tenable to do do those types of analyses. Other ideas welcome.

For starters, @kingjr can you get a good empty-room recording from your setup and put it somewhere for me to get?

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #3631.

This changes our normalization and units to be clearer in `raw.plot_psd`. In the case where `dB=False`, it plots amplitudes instead of power, which seems to be how people prefer to view these things (1/sqrt(Hz)).

I'll paste images of new data when they come up on CircleCI.

~~Not ready for MRG because this will break some `Epochs` plotting. I'm going to wait to fix that until people are happy with the raw plots in case I need to add other things along the way.~~

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -415,7 +416,7 @@ def _plot_lines(data, info, picks, fig, axes, spatial_colors, unit, units,
                     line.set_zorder(2)
                     if spatial_colors is True:
                         line.set_linestyle("--")
-            ax.set_ylabel('data (%s)' % ch_unit)
+            ax.set_ylabel(ch_unit)
```
Comment (agramfort) [inline mne/viz/evoked.py]: why?

Comment (Eric89GXL) [inline mne/viz/evoked.py]: `data ()` doesn't add any meaning

**Comment by agramfort** [comment]
besides LGTM

**Comment by Eric89GXL** [comment]
Existing example:

https://mne-tools.github.io/dev/auto_tutorials/plot_artifacts_correction_filtering.html

Updated version (there are some bugs with the labeling I need to fix):

https://3897-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_artifacts_correction_filtering.html

Newly added example:

https://3897-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/visualization/plot_sensor_noise_level.html

**Comment by Eric89GXL** [comment]
Okay labels are fixed.

Again, for `plot_psd` [old](https://mne-tools.github.io/dev/auto_tutorials/plot_artifacts_correction_filtering.html), [revamped](https://3900-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_artifacts_correction_filtering.html), and [new example](https://3900-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/visualization/plot_sensor_noise_level.html).

For `plot_evoked` minor change [old](https://mne-tools.github.io/dev/auto_tutorials/plot_visualize_evoked.html) and [revamped, no data()](https://3900-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_visualize_evoked.html).

**Comment by agramfort** [comment]
thx @Eric89GXL

**Pull request merged**